src/utils.jl

```diff
@@ -277,12 +277,14 @@ function linetable(arg)
     return ci.linetable::Union{Vector{Core.LineInfoNode},Vector{Any}} # issue #264
     end # @static if
 end
-function linetable(arg, i::Integer; macro_caller::Bool=false)::Union{Expr,LineTypes}
+function linetable(arg, i::Integer; macro_caller::Bool=false)::Union{Expr,Nothing,LineTypes}
```

Comment:
`linetable(arg, ::Integer)` is used in the other places too, like `linenumber(::FrameCode, pc)`, so better to handle the `nothing` case there too?

Reply:
I guess the other option would be to synthesize a special LineNumber node that indicates the absence of information. That way clients that don't care don't have to worry about it

Reply:
I like the `nothing` option since anything that isn't prepared will hard-error.